docs/source/visualization/using_pycmc.ipynb

```diff
@@ -0,0 +1,146 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualizing Web Map Tile Service (WMTS) Layers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here is an example of how to visualize WMTS layers using pyCMC"
```

Comment:
Suggest comma or colon at the end of this line.

Reply:
Agreed and staged for next commit.